src/transport/include/antioch/transport_mixture.h

```diff
@@ -154,14 +170,60 @@ namespace Antioch
 
   template<typename ThermoEvaluator,typename CoeffType>
   inline
-  TransportMixture<ThermoEvaluator,CoeffType>::TransportMixture( const ChemicalMixture<CoeffType>& chem_mix, const ThermoEvaluator & t, const std::string & filename )
+  TransportMixture<ThermoEvaluator,CoeffType>::TransportMixture( const ChemicalMixture<CoeffType>& chem_mix, const ThermoEvaluator & t, 
+                                                                 const std::string & filename, bool verbose, ParsingType type )
     : _chemical_mixture( chem_mix),
       _thermo(t),
       _transport_species(_chemical_mixture.n_species(), NULL )
   {
 
+   ParserBase<CoeffType> * parser(NULL);
+    switch(type)
```

Comment:
Can leave as is for now, but open a new issue to create a `ParserFactory` class that takes care of this and cleans up any other places that have this.